arches_lingo/src/arches_lingo/pages/SchemePage.vue

```diff
@@ -73,7 +79,10 @@ const getRef = (el: object | null, index: number) => {
                     :is="component.component"
                     :ref="(el) => getRef(el, index)"
                     v-bind="component.props"
-                    @open-editor="onOpenEditor(component.id)"
+                    @open-editor="
```

Comment:
The viewer knows what tile the editor needs to work on (edit button per row).  That gets bubbled up as an extra arg.  This is to work in concert with Johnathan's work on the editor for Scheme Labels.  I _could_ just pass up a single extra arg, but that did not seem right - since no other component is doing this yet, and other viewers may need to communicate with editors in other ways.  If you have a better pattern for doing this, let me know.

Reply:
discussed with Chris, we're going to go with just passing a single tileid from viewer to editor.  This makes more sense with #154 coming.